plugins/source/stripe/Makefile

```diff
@@ -5,8 +5,10 @@ build:
 # Test unit
 .PHONY: test
 test: gen-fixtures
-	wget -nc https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.json -P ./resources/testdata
-	go test -race -timeout 3m ./...
+	stripeSDKTag=$$(go list -m github.com/stripe/stripe-go/v74 | awk '{print $$2}') ;\
+	stripeOpenAPITag=$$(curl --silent https://raw.githubusercontent.com/stripe/stripe-go/$$stripeSDKTag/OPENAPI_VERSION) ;\
+	curl --silent https://raw.githubusercontent.com/stripe/openapi/$$stripeOpenAPITag/openapi/spec3.json > ./resources/testdata/spec3.json ;\
```

Comment:
`wget` being replaced by `curl` here...

Reply:
`wget -nc` is "no-clobber" so it won't overwrite if the file already exists... which makes the tests go faster

Reply:
Used `curl` for consistency. We can use `wget` in both. I'd rather overwrite to avoid having a stale file, for example after updating the SDK. Also in CI it doesn't matter

Reply:
Stale file shouldn't happen as it's in `.gitignore`, but one could upgrade the module and then this would create issues... safe to use `curl` unless we put the sdk tag in the filename I guess.

Reply:
> but one could upgrade the module and then this would create issues

Yeah, that's what I meant